charts/keycloak-gatekeeper/templates/deployment.yaml

```diff
@@ -25,7 +25,7 @@ spec:
       serviceAccountName: {{ include "keycloak-gatekeeper.serviceAccountName" . }}
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:{{ .Chart.AppVersion }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.repository | default .Chart.AppVersion }}"
```

Comment:
Shouldn't that be `image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"`   or similar ?

Reply:
Oops. You're right, that's a silly typo!

(And yes, I'm the geekchef ;) )

Reply:
Thanks, didn't catch that